### PR TITLE
#151: fixed Sofortüberweisung selenium tests

### PR DESCRIPTION
**Pull request by andrii-kovalenko-ct**
@butenkor @ahalberkamp 
i've fixed the Sofortüberweisung tests: 

  - The problem was in wrong button search approach, so after the page design change Selenium web-driver found another button (hidden menu button) instead of _submit_ button.
  - also added failsafe check that exactly one submit button exists on the page (see `findSubmitButton()` method)

### CODE REVIEW

**Comment by andrii-kovalenko-ct** [comment]
@lojzatran i heard you have spare time today...
i'd appreciate if you also review it and have any concern (this is not just application specific fix)

File: functionaltests/src/test/java/util/constant/WebDriverCommon.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
  */
 public final class WebDriverCommon {
 
-    public static final String CSS_SELECTOR_SUBMIT = "button[type=submit]";
+    public static final String XPATH_SUBMIT = "//button[@type='submit']";
```
Comment (lojzatran) [inline functionaltests/src/test/java/util/constant/WebDriverCommon.java]: How is this different from the previous solution? If you have 2 submit buttons, it's still not identified properly, right?

File: functionaltests/src/test/java/util/WebDriverSofortueberweisung.java [hunk 2]
```diff
@@ -41,8 +42,10 @@ public WebDriverSofortueberweisung(final String pin, final String tan) {
         timeouts.pageLoadTimeout(DEFAULT_TIMEOUT, TimeUnit.SECONDS);
         timeouts.setScriptTimeout(DEFAULT_TIMEOUT, TimeUnit.SECONDS);
 
+        manage().window().fullscreen();
```
Comment (lojzatran) [inline functionaltests/src/test/java/util/WebDriverSofortueberweisung.java]: What's the reason of this?

Comment (andrii-kovalenko-ct) [inline functionaltests/src/test/java/util/WebDriverSofortueberweisung.java]: In some cases it is possible that button control is hidden in a virtual browser window, and selenium will complain the button can't be clicked! 

This is not a case right now, but it protects us from future fails if they change design and the button is located on the bottom or at the right.

File: functionaltests/src/test/java/util/constant/WebDriverCommon.java (hunk 1 above)
Comment (andrii-kovalenko-ct) [inline functionaltests/src/test/java/util/constant/WebDriverCommon.java]: correct, we would need additional logic for such a case (for instance, select by ID, name or class), but it is very unexpected right now that sofortüberweisung will have such a design.

**Comment by lojzatran** [comment]
Do you have task number so I can log time?

**Pull request merged**